src/nunit.analyzers/DisposeFieldsAndPropertiesInTearDown/DisposeFieldsAndPropertiesInTearDownAnalyzer.cs

```diff
@@ -149,6 +149,22 @@ private static void AnalyzeDisposableFields(SyntaxNodeAnalysisContext context)
                 disposeMethods = disposeMethods.Union(value.Split(AdditionalDisposalMethodsSeparators, StringSplitOptions.RemoveEmptyEntries));
             }
 
+            // Sometimes virtual method are called from base class SetUp/TearDown methods.
+            // Allow the users to specify these as well.
+            ImmutableHashSet<string> additionalSetUpMethods = ImmutableHashSet<string>.Empty;
+            ImmutableHashSet<string> additonalTearDownMethods = ImmutableHashSet<string>.Empty;
+
+            if (options.TryGetValue("dotnet_diagnostic.NUnit1032.additional_setup_methods", out value))
```

Comment:
This mean that all such methods in the project should follow the same naming convention, right?